app/assets/stylesheets/layout_components/subscriptions.scss

```diff
@@ -0,0 +1,20 @@
+.subscriptions {
+  .row {
+    padding: 15px;
+
+    &.section-heading {
+      padding-left: 10px;
+      background-color: #eeeeee;
```

Comment:
for all the colours in this file can you please change them to use our variable names for those colours? They are all available from our style guide http://ama-layout.herokuapp.com/